sormas-api/src/main/java/de/symeda/sormas/api/caze/CaseLogic.java

```diff
@@ -53,15 +55,21 @@ public static void validateInvestigationDoneAllowed(CaseDataDto caze) throws Val
 		}
 	}
 
-	public static Date getStartDate(Date onsetDate, Date reportDate) {
+	public static FollowUpPeriodDto getStartDate(CaseDataDto caseDto, List<SampleDto> samples) {
```

Comment:
`getStartDate` is not a logic that is only used for follow-up calculations. It determines the date at which the case became relevant, which still is either the onset date or the report date. The earliest collected sample should not be part of this logic. This probably calls for a separate helper method (e.g. `calculateFollowUpStart`) that is only used in the follow-up context